patches/api/0468-Added-EntityDamageDoorEvent-and-implemented-into-Ser.patch

```diff
@@ -0,0 +1,123 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: CMarcoo <cmarco.org@gmail.com>
+Date: Thu, 14 Mar 2024 17:04:43 +0100
+Subject: [PATCH] Added EntityDamageDoorEvent and implemented into Server.
+
+
+diff --git a/src/main/java/io/papermc/paper/event/entity/EntityDamageDoorEvent.java b/src/main/java/io/papermc/paper/event/entity/EntityDamageDoorEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..43614d7916be316710d2fe848b3b60187cca978f
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/event/entity/EntityDamageDoorEvent.java
+@@ -0,0 +1,111 @@
++package io.papermc.paper.event.entity;
++
++import com.google.common.base.Preconditions;
++import org.bukkit.block.data.BlockData;
++import org.bukkit.entity.Entity;
++import org.bukkit.event.Cancellable;
++import org.bukkit.event.HandlerList;
++import org.bukkit.event.entity.EntityEvent;
++import org.jetbrains.annotations.NotNull;
++import org.jetbrains.annotations.Range;
++
++/**
++ * Called for every tick an entity is damaging a door.
++ * <p>
++ * This event is only called when a new level of progress
++ * has been reached by the damaging entity.
++ */
++public class EntityDamageDoorEvent extends EntityEvent implements Cancellable {
++
```

Comment:
So the event is called every tick OR only when new damage progress level is displayed? This is a bit unclear, at least for me